super-intel.c

```diff
@@ -5976,12 +5976,12 @@ static int add_to_super_imsm(struct supertype *st, mdu_disk_info_t *dk,
 			     unsigned long long data_offset)
 {
 	struct intel_super *super = st->sb;
```

Comment:
a bit of inconsistency appeard due to the use of MDADM_STATUS_ERROR, the function still returns int, I suspect because it is used as a pointer to a global function, at the same time from one place further int is returned "return add_to_super_imsm_volume(st, dk, fd, devname);"
would you be able to standardize this in the context of add_to_super?

Reply:
not in this patch, sorry :(
using enum va;ues for int function is not an issue.